nginx.conf

```diff
@@ -24,9 +24,30 @@ http {
    access_log /dev/stdout;
    root /app;
 
-   location / {
+   location /business {
      alias /app;
      try_files $uri /index.html;
+     gzip on;
+     gzip_vary on;
+     gzip_min_length 10240;
+     gzip_proxied any;
+     gzip_types text/plain text/css text/xml text/javascript application/x-javascript application/xml;
+     gzip_disable "MSIE [1-6]\.";
+   }
+
+   # For status of ngnix service, OpenShift is configured to call this
```

Comment:
Can this be used in other 2 UIs as well?

Reply:
Done.

Reply:
Awesome, thanks!